src/org/openlcb/can/NIDaAlgorithm.java

```diff
@@ -88,6 +88,13 @@ public void processFrame(OpenLcbCanFrame f) {
             return; // as a convenience, ignore
         }
 
+        if (f.isAliasMapDefinition()) {
```

Comment:
Shouldn't this be testing for enquiry?
Also we need to verify that the payload of the enquiry is empty or matches our node ID.

Reply:
Thanks.  Done, I think.